README.md

```diff
@@ -0,0 +1,72 @@
+# IOXIO Dataspace example app
+
+This demo application is designed to show a practical example on how to create an
+application using Dataspace tecnhologies.
+
+It consists of simple Python backend that is responsible for authentication and data
+retrieval and React-based frontend application.
+
+You can check the [online demo](TODO: add a link there) or run it locally (this way
+requires some configuration, so check [configuration](#configuration) section below).
+
+Main idea is to demostrate how to:
+
+- Perform authentication in a dataspace
+- Handle application configuration correctly
```

Comment:
This is not the purpose of the application. People can handle their configuration however they want.